### PR TITLE
10.1016/j.cell.2021.01.053 Covid PBMC dataloader

### PR DESCRIPTION
**Pull request by lauradmartens**
Added the data loader for the publication [COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas](https://www.sciencedirect.com/science/article/pii/S0092867421001483?via%3Dihub).

Two things:
* The adata download link is from Google Drive, I am not sure if that works with your automatic download. 
* One enhancement would be to add more fine-grained cell types according to the obs key `celltype`

### CODE REVIEW

File: sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+dataset_structure:
+    dataset_index: 1
+    sample_fns:
+        - "COVID19_ALL.h5ad"
+dataset_wise:
+    author: "Ren, Xianwen"
+    default_embedding: "NA"
```
Comment (davidsebfischer) [inline sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml:7]: Is there no UMAP or tsne in this object? If yes, you can just indicate the corresponding obsm key here, if this object does not have any just leave blank.

**Review by davidsebfischer — CHANGES_REQUESTED** [review]

File: sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml (hunk 1 above)
Comment (lauradmartens) [reply sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml:7]: The dataset has its embedding in `adata.obs.umap1` and `adata.obs.umap2` but I could put it to an obsm key in the load function. `create-dataloader`  fills `default_embedding` with  `"NA"` by default, maybe that should be changed then?

Comment (davidsebfischer) [reply sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml:7]: Yes, @Zethson let s add this to #370

**Review by davidsebfischer — APPROVED** [review]

**Comment by davidsebfischer** [comment]
Thank you @lauradmartens!

File: sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    dataset_index: 1
+    sample_fns:
+        - "COVID19_ALL.h5ad"
+dataset_wise:
+    author: "Ren, Xianwen"
+    default_embedding: "X_tsne"
+    doi_preprint:
+    doi_journal: "10.1016/j.cell.2021.01.053"
+    download_url_data: "https://drive.google.com/file/d/1IwWcn4W-YKgNbz4DpNweM2cKxlx1hbM0/view"
+    download_url_meta: "None"
```
Comment (davidsebfischer) [inline sfaira/data/dataloaders/loaders/d10_1016_j_cell_2021_01_053/human_na_2021_na_ren_001.yaml:11]: @lauradmartens was this "None" introduced by the CLI? If so we need to take it out, this field should be empty if there is no metadata URL.

**Pull request merged**